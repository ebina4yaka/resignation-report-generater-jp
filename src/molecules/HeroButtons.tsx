import React from 'react'
import Grid from '@material-ui/core/Grid'
import Button from '@material-ui/core/Button'
import { makeStyles } from '@material-ui/core/styles'
import Link from 'next/link'

const useStyles = makeStyles((theme) => ({
  root: {
    marginTop: theme.spacing(4),
  },
}))

export default function HeroButtons(): React.ReactElement {
  const classes = useStyles()
  return (
    <div className={classes.root}>
      <Grid container spacing={2} justify="center">
        <Grid item>
          <Link href="/create" passHref>
            <Button variant="contained" color="primary">
              退職届を作成する
            </Button>
          </Link>
        </Grid>
      </Grid>
    </div>
  )
}
