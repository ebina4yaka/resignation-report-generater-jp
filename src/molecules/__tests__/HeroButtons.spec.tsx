import renderer from 'react-test-renderer'
import HeroButtons from '../HeroButtons'

jest.mock('next/link', () => 'MockedLink')

test('Snapshot test for HeroButtons', () => {
  const tree = renderer.create(<HeroButtons />).toJSON()
  expect(tree).toMatchSnapshot()
})
