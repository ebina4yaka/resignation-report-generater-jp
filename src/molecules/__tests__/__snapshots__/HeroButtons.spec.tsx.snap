// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for HeroButtons 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <MockedLink
      href="/about"
      passHref={true}
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-1 MuiButton-containedSecondary"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          使い方
        </span>
      </button>
    </MockedLink>
    <MockedLink
      href="/create"
      passHref={true}
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-1 MuiButton-containedPrimary"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          作成する
        </span>
      </button>
    </MockedLink>
  </div>
</div>
`;
